### PR TITLE
Ef/fetch geocoded constituents

### PR DESCRIPTION
**Pull request by emilyfeder**
Emily R and I worked on this endpoint which gets all member data and all constituent coordinates fields and then merges the two responses so as to reduce the number of api calls we are making. This should set up Noah to get all the data he needs for the front end. Next steps are to add a test and to refactor a bit to make this DRYer.

### CODE REVIEW

**Comment by alexsoble** [comment]
I think this looks great! I'll leave some questions/suggestions, but don't let these slow you down if you're on a roll!

File: jcua_renxt/blueprints/hello_world/__init__.py [hunk 1]
```diff
@@ -68,3 +68,31 @@ def geocode_all_active_constituents():
...
 
+
+@hello_world.route('/fetch_geocoded_constituents')
+def fetch_geocoded_constituents():
+    members_endpoint = 'constituent/v1/constituents?constituent_code=Member'
+    members_endpoint += '&limit=3000'  # Avoiding writing code to handle pagination
+    members_response = renxt.get(members_endpoint)
+    members = members_response.json()['value']
+    
+    custom_fields_endpoint = 'constituent/v1/constituents/customfields?category=Coordinates'
```
Comment (alexsoble) [inline jcua_renxt/blueprints/hello_world/__init__.py]: How long does it take to execute these two queries in sequence? Depending on how long it takes, do we think it's worthwhile to try to run the queries in parallel?

File: jcua_renxt/blueprints/hello_world/__init__.py [hunk 2]
```diff
@@ -68,3 +68,31 @@ def geocode_all_active_constituents():
...
+    custom_fields_endpoint += '&limit=3000'
+    custom_fields_response = renxt.get(custom_fields_endpoint)
+    custom_fields = custom_fields_response.json()['value']
+    
+    coordinates_lookup = {}
+    for custom_field in custom_fields:
+        coordinates_lookup[custom_field.get('parent_id')] = custom_field.get('value')
+    
+    member_data = [{
```
Comment (alexsoble) [inline jcua_renxt/blueprints/hello_world/__init__.py]: The logic involved in building up this `member_data` variable looks like a good candidate for a unit test.

**Comment by alexsoble** [comment]
@emilyfeder I realized that I missed the most important question yesterday! 

Were you able to 🔌 in the front-end map to this endpoint? Did it successfully populate the map with data?

If so, I wouldn't worry too much about adding tests and refactoring to this endpoint — it's not super gnarly or complex code. My vote would be ship first.

**Comment by Noleli** [comment]
Haven’t tried yet. Hoping it Just Works™!

**Pull request merged**